### ISSUE TITLE
Documentation for different parts

### ISSUE THREAD

**Issue by parunach** [ipfs/py-ipfs-api#94]
I see that the simple writeup is only for connecting to daemon and adding a file.

Please add for 
-How to list all the files added with different methods
-How to get a specific file with multiple ways

Without this, the usability is insufficient.

**Comment by alexander255** [comment]
Yes, *someone* needs to write more documentation: Particularly a tutorial, which is basically what you are implying, would be a very nice addition to our current documentation indeed!

A short description of the `.id()` output may be found here, but it's not very complete either:
https://ipfs.io/docs/commands/#ipfs-id

**Comment by parunach** [comment]
Yes, it will be great if someone does this. I see no books on ipfs, makes it hard to understand the details. All references are to ipfs website, GitHub and some articles.

**Comment by lordcirth** [comment]
Would this documentation be about IPFS, or just the use of this python API?
